api-metastore/src/main/java/org/zalando/nakadi/controller/advice/SchemaExceptionHandler.java

```diff
@@ -24,4 +27,20 @@
         AdviceTrait.LOG.debug(exception.getMessage());
         return create(Problem.valueOf(NOT_FOUND, exception.getMessage()), request);
     }
+
+    @ExceptionHandler(SchemaEvolutionException.class)
+    public ResponseEntity<Problem> handleSchemaEvolutionException(
+            final SchemaEvolutionException exception,
+            final NativeWebRequest request) {
+        AdviceTrait.LOG.debug(exception.getMessage());
+        return create(Problem.valueOf(UNPROCESSABLE_ENTITY, exception.getMessage()), request);
+    }
+
+    @ExceptionHandler(SchemaValidationException.class)
+    public ResponseEntity<Problem> handleSchemaEvolutionException(
```

Comment:
Shouldn't the name of this method be `handleSchemaValidationException`?

Reply:
True, fixed.